src/api/middleware/validators/offer.js

```diff
@@ -53,7 +53,20 @@ const create = useExpressValidators([
 
     body("jobMaxDuration", ValidationReasons.DEFAULT)
         .optional()
-        .isInt().withMessage(ValidationReasons.INT),
+        .isInt().withMessage(ValidationReasons.INT)
+        .custom((jobMaxDuration, { req }) => {
+            try {
+                const { jobMinDuration } = req.body;
+
+                if (jobMinDuration >= jobMaxDuration) {
```

Comment:
You need to also check if jobMinDuration is defined before comparing. If it is not defined (since it's optional) this validation should pass

Add `if(!jobMinDuration) return true;` before that check